AirportTicketBooking/ManagerFactory.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+
+namespace AirportTicketBooking
+{
+    public class ManagerFactory
+    {
+        public void CreateNewManager()
+        {
+            var rnd = new Random();
+            var csvio = new CSVIO();
+            
```

Comment:
You should not create a dependency in your class, let the caller of your class be responsible of creating and passing them

 ```   
    public class ManagerFactory
    {
        private Random _random;
        private CSVIO _csvio;

        public ManagerFactory(Random random, CSVIO csvio)
        {
            _random = random;
            _csvio  = csvio;
        }
   }
```

Reply:
Done.